concent_api/concent_api/settings/development.py

```diff
@@ -17,8 +17,4 @@
     "admin-panel",
 ]
 
-RAVEN_CONFIG = {
-    'dsn': 'https://',
-}
-
```

Comment:
OK, but what about the warnings?

Reply:
There is no warning, the `sentry` only adds information that the `raven` is not configured.

Reply:
OK. I thought it was a warning. We really want to avoid having warnings when there is no issue because it teaches people to ignore them.